cirq/google/calibration/phased_fsim.py

```diff
@@ -254,8 +254,8 @@ class FloquetPhasedFSimCalibrationOptions:
     characterize_gamma: bool
     characterize_phi: bool
 
-    @staticmethod
-    def all_options() -> 'FloquetPhasedFSimCalibrationOptions':
+    @classmethod
+    def with_all_angles_characterization(cls) -> 'FloquetPhasedFSimCalibrationOptions':
```

Comment:
You can `return cls(...)`

BUT! Do we need these methods at all? I see you've defined a global variable for these two configurations that calls this factory method, but why not explicitly construct these two objects and delete the two factory functions. 

```python
ALL_ANGLES_CHARICTERIZATION = FloquetPhasedFsimCalibrationOptions(true true true true)
WITHOUT_CHI_CHARICTERIZATION = FloquetPhaseFsimCalibrationOptions(true true true false)
```

Reply:
Done.